### PR TITLE
Drupal8 content management platform for building a website. This plug…

### PR DESCRIPTION
**Pull request by SumGuyV5**
…-in uses apache24, mysql, php72 and drush.

### CODE REVIEW

**Comment by kmoore134** [comment]
Took a look at this, appears functional on 11.3-RC1. One small nit, but not specific to this pull-request, is here:

```
+ echo 'drupal8 now installed.\n'
+ echo '\nYour MySQL Root password is "551afdbbd60609f21471".\n'
+ echo '\nYour Drupal Database password is "998a2afc1e380856a843".\n'
+ echo '\nDrupal Admin user Password is "Gv99DYpoHi".\n'
```
https://github.com/SumGuyV5/iocage-plugin-drupal8/blob/master/post_install.sh#L137

When using "echo" with "\n" on FreeBSD, you would use `echo -e`

Additionally, you have a single > redirect on those lines, which is write-over mode, not append. It should be:

`echo "\nYour MySQL Root password is \"${MYSQL_ROOT_PASS}\".\n" >> /root/PLUGIN_INFO`

**Comment by kmoore134** [comment]
@SumGuyV5 Ping! Did you get a chance to take a look at the comments for the post_install script?

**Comment by SumGuyV5** [comment]
Yes the post_install script has been fix. Thank for the feed back and help.

File: drupal8.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "drupal8",
+  "plugin_schema": "2",
+  "release": "11.3-RELEASE",
+  "artifact": "https://github.com/SumGuyV5/iocage-plugin-drupal8.git",
+  "pkgs": [
+    "www/apache24",
```
Comment (miwi-fbsd) [inline drupal8.json]: Can you please remove all categories ?

File: drupal8.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "databases/mysql56-client",
+    "www/drupal8",
+    "databases/php72-mysqli",
+    "security/php72-hash",
+    "graphics/php72-gd",
+    "databases/php72-pdo_mysql",
+    "www/mod_php72",
+    "drush-php72"
+  ],
+  "packagesite": "http://pkg.FreeBSD.org/FreeBSD:11:amd64/release_3",
```
Comment (miwi-fbsd) [inline drupal8.json]: Would it not make sense to switch this to latest? To ensure the latest update will be catch? Drupal is know for they Security issues.

**Comment by miwi-fbsd** [comment]
Thanks, I will test it right now and let you know.

**Comment by miwi-fbsd** [comment]
Once https://github.com/SumGuyV5/iocage-plugin-index/pull/2 is in we can merge it in :-).

**Pull request merged**